### ISSUE TITLE
agGridColumn: missing column properties

### ISSUE THREAD

**Issue by mroseboom** [ceolter/ag-grid-aurelia#25]
**I'm submitting a ...**
```
[X] bug report => search github for a similar issue or PR before submitting
[ ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/ceolter/ag-grid-aurelia/blob/master/CONTRIBUTING.md#question
```
**Current behavior**
agGridColumn does not support the following properties:
 - suppressToolPanel
 - suppressKeyboardEvent(params)

**Expected behavior**
agGridColumn should support those attributes

**What is the motivation / use case for changing the behavior?**
Should be able to use the built-in column properties on the `ag-grid-column` component.

**Please tell us about your environment:**

* **ag-grid-aurelia version:** 16.0.0
* **ag-Grid/ag-Grid-Enterprise version:** 16.0.0
* **Aurelia version:** 16.0.0

* **Browser:** all

* **Language:** all

**Comment by ShaunVanDyk** [comment]
Is rowDrag also missing? This was added in v16: https://www.ag-grid.com/javascript-grid-row-dragging/#managed-dragging

**Comment by mroseboom** [comment]
@ShaunVanDyk You're right, the property `rowDrag` is also missing. I'll add that one too.

**Comment by mroseboom** [comment]
Added both new properties rowData and toolPanelClass in the pull-request

**Comment by avrahamcool** [comment]
looks like `lockPinned` is also missing.

https://www.ag-grid.com/javascript-grid-pinning/

**Comment by mroseboom** [comment]
@avrahamcool `lockPinned` is added.
